Makefile

```
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REGISTRY                                   := eu.gcr.io/gardener-project/gardener
+REGISTRY                                   := europe-docker.pkg.dev/gardener-project/releases/gardener
```

```suggestion
REGISTRY                                   := europe-docker.pkg.dev/gardener-project/snapshots/gardener
```

See https://github.com/gardener/gardener/pull/8968#discussion_r1431903900.

---

@timuthy : thanks, will give it another try (IIRC, there were test-errors if Makefile diverged from skaffold-files)